src/backend/bacon-video-widget-gst-missing-plugins.c

```diff
@@ -59,53 +59,6 @@ typedef struct
 }
 TotemCodecInstallContext;
 
-#ifdef GDK_WINDOWING_X11
```

Comment:
Okay, updated this to separate out the commits adding the new API and adding the VLC path. Kept the old PK code in place, except that we don't call it anymore.